composer.json

```diff
@@ -19,29 +19,31 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4 || ^8.0",
 
-        "sylius/sylius": "~1.9.0",
+        "sylius/sylius": "1.9.*",
         "friendsofsymfony/jsrouting-bundle": "^2.2",
-        "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev"
+        "symfony/flex": "^1.10",
+        "symfony/asset": "^4.4 || ^5.2",
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev"
```

Comment:
Might be considered as dev-dependency only (Sylius installations will have it defined in the root package if needed).